alf/utils/checkpoint_utils.py

```diff
@@ -348,5 +352,32 @@ def save(self, global_step):
         torch.save(optimizer_state, f_path + '-optimizer')
         torch.save(replay_buffer_state, f_path + '-replay_buffer')
 
+        if self._global_step == -1:
+
+            def _dummy_value(v):
```

Comment:
map_structure will always reach a leaf node. so this check is unnecessary

Reply:
Changed